hnn_core/gui/gui.py

```diff
@@ -164,12 +463,42 @@ def __init__(self, theme_color="#8A2BE2", log_window_height="100px",
                                                  value='Joblib',
                                                  description='Backend:')
 
+        # analysis tab
+        analysis_style = {'description_width': '200px'}
+        self.widget_if_save_figures = Dropdown(options=[('Yes', True),
+                                                        ('No', False)],
+                                               value=False,
+                                               description='Save figures:',
+                                               style=analysis_style)
+        self.widget_if_save_spectral_data = Dropdown(
+            options=[('Yes', True), ('No', False)], value=False,
+            description='Save Spectral Data:', style=analysis_style)
+
+        self.widget_max_spectral_frequency = FloatText(
+            value=100.0, description='Max Spectral Frequency (Hz):',
+            disabled=False, style=analysis_style)
+        self.widget_dipole_scaling = FloatText(value=3000.0,
+                                               description='Dipole Scaling:',
+                                               disabled=False,
+                                               style=analysis_style)
+        self.widget_dipole_smooth = FloatText(
+            value=30.0,
+            description='Dipole Smooth Window (ms):',
+            disabled=False, style=analysis_style)
+        self.widget_if_record_somatic_v = Dropdown(
+            options=[('Yes', True), ('No', False)], value=False,
+            description='Record Somatic Voltages:', style=analysis_style)
+
```

Comment:
This doesn't seem to be related to "analysis" ... you need to specify at the beginning of the simulation

Reply:
This is from the old gui. I can move it